File: python/Programmers_level/level1/두 정수 사이의 합.py
def solution(a, b):
    answer = 0
    
    # 예외처리
    if a > b:
        a, b = b, a
    
    for i in range(a, b+1):
        answer += i
    # answer = sum(range(a, b+1)) 이렇게도 가능!
        
    return answer

File: Javascript/Programmers_level/level1/두 정수 사이의 합.js
function solution(a, b) {
  let answer = 0;
  if (a > b) {
    let temp;
    temp = b;
    b = a;
    a = temp;
  }
  for (let i = a; i < b + 1; i++) {
    answer += i;
  }
  return answer;
}
